sympy/utilities/tests/test_lambdify.py

```diff
@@ -1891,3 +1891,64 @@ def test_assoc_legendre_numerical_evaluation():
 
     assert all_close(sympy_result_integer, mpmath_result_integer, tol)
     assert all_close(sympy_result_complex, mpmath_result_complex, tol)
+
+def test_derivative_issue_26404():
+    r'''
+        test issue fixed when using cse in lambdify when some arguments
+        are Derivatives and they appear in the expression
+    '''
+    from sympy import (cos, sin, Matrix, symbols)
+    from sympy.physics.mechanics import (dynamicsymbols)
+    t = symbols("t")
+    x = Function("x")(t)
+    xd = x.diff(t)
+    xdd= xd.diff(t)
+    assert lambdify((xd, x), xd, cse=True)(1, 1) ==1
+    assert lambdify((xd, x), xd + x, cse=True)(1, 1) ==2
```

Comment:
Same for all.
```suggestion
    assert lambdify((xd, x), xd + x, cse=True)(1, 1) == 2
```

Reply:
yes you're right, I'll put it in the next commit too!